amundsen_application/static/js/pages/DashboardPage/index.tsx

```diff
@@ -181,12 +193,12 @@ export class DashboardPage extends React.Component<
             />
             <div className="body-2">
               Dashboard in&nbsp;
+              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
if anyone has better suggestion to solve this, please do. I look at the link for the eslint which suggests to use a button which I don't think it matches our UX design suggestion.

Reply:
Yeah, it should be a button.
Try something like this:
```
<button type="button" className="clickable-group btn btn-link" id="dashboard-group-link" onClick={this.searchGroup} >{dashboard.group_name}</button>

Reply:
You wouldn't need the clickable-group class

Reply:
Also, if the id is just for tracking purposes, you should use 'data-type="dashboard-group-link"' instead

Reply:
cool, thanks @Golodhros . pr updated